app/helpers/repositories_helper.rb

```diff
@@ -194,14 +194,11 @@ def scm_select_tag(repository)
     select_tag('repository_scm',
                options_for_select(scm_options, repository.class.name.demodulize),
                disabled: (repository && !repository.new_record?),
-               onchange: remote_function(
-                  url: {
-                      controller: '/repositories',
-                      action: 'edit',
-                      id: @project
-                        },
-               method: :get,
-               with: "Form.serialize(this.form)")
+               data: { remote: true,
+                       url: url_for(controller: "repositories",
+                                    action: "edit",
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.